src/utils/helpers/servicesHelpers.ts

```diff
@@ -68,8 +69,10 @@ export const addUniqueIdInObjects = (objects: ObjectType[]): any[] => {
   return objects.map((object: ObjectType): ObjectType => ({ ...object, id: uuidv4() }));
 };
 
-export const getErrorMessage = (errorObject: AxiosError<any>): string => {
-  return errorObject.response?.data.message || errorObject.message;
+export const getErrorMessage = (errorObject: AxiosError<any>, language: string): string => {
+  return language === Language.Ru
+    ? 'При загрузке что-то пошло не так...'
+    : errorObject.response?.data.message || errorObject.message;
 };
```

Comment:
Did you mean this?
```
export const getErrorMessage = (errorObject: AxiosError<any>, language: string): string => {
  return language === Language.Ru
    ? i18next.t(ErrorTxtKey.SomeError)
    : errorObject.response?.data.message || errorObject.message;
};
```

In this case I can not use the full i18next logic, because I expect a new English text every time, which I receive from the thrown error.